clients/gitlabrepo/statuses_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gitlabrepo
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/xanzy/go-gitlab"
+
+	"github.com/ossf/scorecard/v4/clients"
+)
+
+func Test_listStatuses(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name         string
+		responsePath string
+		want         []clients.Status
+		wantErr      bool
+	}{
+		{
+			name:         "valid webhook",
+			responsePath: "./testdata/valid-status",
+			want: []clients.Status{
+				{
+					State:     "pending",
+					Context:   "bundler:audit",
+					URL:       "https://gitlab.example.com/janedoe/gitlab-foss/builds/91",
+					TargetURL: "https://gitlab.example.com/janedoe/gitlab-foss/builds/91",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:         "invalid webhook",
```

Comment:
copy/paste typo, but also see comment below on test name.

what do you think about "handles error listing statuses" or something like that?

Reply:
👍